resources/js/router/views/WorkshopView/index.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+    <div class="container py-3">
+        <div>
+            <AttendanceCard/>
```

Comment:
Raima, this line is what was producing the JS console error! Just remove this line and you'll be good to go. 👍